eclair-core/src/main/scala/fr/acinq/eclair/payment/Bolt11Invoice.scala

```diff
@@ -89,9 +89,9 @@ case class Bolt11Invoice(prefix: String, amount_opt: Option[MilliSatoshi], creat
 
   lazy val relativeExpiry: FiniteDuration = FiniteDuration(relativeExpiry_opt.getOrElse(DEFAULT_EXPIRY_SECONDS), TimeUnit.SECONDS)
 
-  lazy val minFinalCltvExpiryDelta: Option[CltvExpiryDelta] = tags.collectFirst {
+  lazy val minFinalCltvExpiryDelta: CltvExpiryDelta = tags.collectFirst {
```

Comment:
Nit: Can you make `fallbackAddress`, `relativeExpiry_opt` and `minFinalCltvExpiryDelta` all on a single line? I'm OCDing on the inconsistency.

Reply:
Done, I've actually also removed  `relativeExpiry_opt`